osf-packager/src/test/resources/json/IpmPackagerTest/testCreatePackageWithWiki/nodes/ng9em/comments/index.json

```diff
@@ -1,142 +0,0 @@
-{
```

Comment:
Regarding the test files, would they have been moved from the `IpmPackagerTest` to `OsfContentProviderTest` as a result of a refactoring of your IDE?  It's always a good idea to preview your diff locally before submitting a PR, that way there aren't any surprises.

I imagine that the moved files are OK: that is, they were necessary for `IpmPackager` tests prior to the refactoring, so they will need to be refactored (e.g. moved to a new directory) so they continue to work against `OsfContentProvider`.  The to-be-developed test plan can determine whether or not the tests in `OsfContentProviderTest` stay, or if they get refactored into a more general test harness/fixture.